tools/go-cli/go-whisk/whisk/shared.go

```diff
@@ -38,4 +38,5 @@ type Parameters *json.RawMessage
 type Limits struct {
     Timeout int `json:"timeout,omitempty"`
     Memory  int `json:"memory,omitempty"`
+    Logsize int `json:"logs"`
```

Comment:
Cheers for pointing that out. We haven't tried that. The unfortunate is: `0` is overloaded as the "empty" value for `int`. Investigating...